src/LintingViewExtension/Docs/LinterViewHelpDoc.html

```diff
@@ -0,0 +1,185 @@
+﻿<!--
```

Comment:
Let's follow the pattern introduced in the Documentation Browser PR for how we handle our `.html` doc files: 
https://github.com/DynamoDS/Dynamo/pull/10343


So this file should move to `src/DocumentationBrowserViewExtension/Docs/LinterViewHelpDoc.html`

... and then embed the file in the DocumentationBrowserExtension.dll by adding a reference here: https://github.com/DynamoDS/Dynamo/blob/e736f30095cc2b554008f6b1e1fa762244c63e33/src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.csproj#L159

I can come back and replace the lorem ipsum in a follow-up PR.

Reply:
Technically there shouldnt be any need to put them into the DocumentationBrowsers `doc` folder, think that is mostly for all the expanded error messages and some generic content (at least it was like that back when we did the doc browser). 

IMO it makes sense to keep the help html's alongside what they are intended to describe or help with. We did the same for the python 3 stuff https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/PythonNodeModels/Docs/PythonMigrationWarning.html

Can easily move it if you prefer, just think its nicer to have it like this.